mysql-test/t/percona_bug_ps3940.test

```diff
@@ -0,0 +1,17 @@
+# PS-3940: Warnings/errors when switching from the memory engine to innodb
```

Comment:
No need to add a new test file - extend the most suitable one of percona_heap_* tests